nominal/core/client.py

```diff
@@ -563,6 +566,36 @@ def create_video_from_mcap_io(
             raise NominalIngestError("No or invalid video RID returned")
         return self.get_video(response.outputs[0].target.video_rid)
 
+    def create_asset(
+        self,
+        name: str,
+        description: str | None = None,
+        *,
+        properties: Mapping[str, str] | None = None,
+        labels: Sequence[str] = (),
+    ) -> Asset:
+        """Create an asset."""
+        request = scout_asset_api.CreateAssetRequest(
+            description=description,
+            labels=list(labels),
+            properties={} if properties is None else dict(properties),
+            title=name,
+            attachments=[],
+            data_scopes=[],
```

Comment:
This is the most important part of creating an asset as it defines the underlying data. Would it be possible to implement this functionality?

Reply:
@ad-nmnl how do you imagine `data_scopes` usage here? Is this like `ref_name`?